napari/_qt/qt_main_window.py

```diff
@@ -1142,6 +1143,7 @@ def _update_theme(self, event=None):
 
             self._qt_window.setStyleSheet(get_stylesheet(value))
 
+    @ensure_main_thread
```

Comment:
```suggestion
    @ensure_object_thread
```

Reply:
These are methods of `Window`, which is not a `QObject`, so this doesn't work.

Reply:
my bad